### PR TITLE
Add scipy/numpy/sklearn example for IRIS

### PR DESCRIPTION
**Pull request by busche**
Even though the initial example project is great for getting started, a more complex example project is needed to gain more insight into how pygradle can be configured and used.
This example project uses the classical text book example for the IRIS dataset classification task in order to train and deploy a SVM model.

### CODE REVIEW

File: examples/iris-classification/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+You are now ready to run your build, using
+
+```
+$ ./gradlew build
+```
+
+Then, after a longish build run, you should be able to call the executable in order to see the output which should be similar to this:
+
+	root@64e86ed8927a:/tmp/working/pygradle_busche/examples/iris-classification# ./build/deployable/bin/classify_iris
```
Comment (sixninetynine) [inline examples/iris-classification/README.md]: please remove the full path here, relative paths are fine for the example

File: examples/iris-classification/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	Accuracy score on the IRIS dataset using a 60/40 split: 0.966666666667
+	root@64e86ed8927a:/tmp/working/pygradle_busche/examples/iris-classification#
```
Comment (sixninetynine) [inline examples/iris-classification/README.md]: same as above

File: examples/iris-classification/README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+You are now ready to run your build, using
+
+```
+$ ./gradlew build
+```
+
+Then, after a longish build run, you should be able to call the executable in order to see the output which should be similar to this:
+
+	root@64e86ed8927a:iris-classification# ./build/deployable/bin/classify_iris
```
Comment (sixninetynine) [inline examples/iris-classification/README.md]: can you remove this prompt (the `root@64e86ed8927a:iris-classification#` portion) and just replace it with `$` to indicate a shell prompt?

File: examples/iris-classification/README.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+	Accuracy score on the IRIS dataset using a 60/40 split: 0.966666666667
+	root@64e86ed8927a:iris-classification#
```
Comment (sixninetynine) [inline examples/iris-classification/README.md]: this line can be omitted

Comment (sixninetynine) [inline examples/iris-classification/README.md]: specifically i mean line 45 can be omitted

**Comment by sixninetynine** [comment]
@busche sorry to nitpick but can you squash your commits? thanks in advance!

File: examples/iris-classification/test/test_dataset.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
+'''
+Created on 24.11.2016
+
+@author: busche
```
Comment (ethankhall) [inline examples/iris-classification/test/test_dataset.py]: Please remove the @author tag.

File: examples/iris-classification/pinned.txt [hunk 6]
```diff
@@ -0,0 +1,6 @@
+numpy==1.11.2
```
Comment (ethankhall) [inline examples/iris-classification/pinned.txt]: This whole file should be auto generated, can we remove it and add it to the gitignore?

**Comment by busche** [comment]
I'll squash the commits once any other changes are integrated.

**Comment by busche** [comment]
re-opening a squashed PR as #91

**Pull request closed without merging**